src/search/search.component.html

```diff
@@ -0,0 +1,41 @@
+<div
+	class="bx--search"
+	[ngClass]="{
+		'bx--search--sm': size === 'sm',
+		'bx--search--xl': size === 'xl',
+		'bx--search--light': theme === 'light',
+		'bx--skeleton': skeleton,
+		'bx--toolbar-search': toolbar,
+		'bx--toolbar-search--active': toolbar && active
+	}"
+	role="search">
+	<label class="bx--label" [for]="id">{{label}}</label>
```

Comment:
Also support templates?

Reply:
Don't think this is an actual label? Looking at [`carbon-design system`](https://www.carbondesignsystem.com/components/search/code) I don't see any label